### PR TITLE
[FIX] Clarify name of "BrainVision" format

### LINKED ISSUE #172: iEEG specification for BrainVision format
In the iEEG specification, BrainVision is one format accepted but the chosen extensions are only .eeg, .vhdr, and .vmrk. However, in this format, it is possible to save the data with the extensions .dat, .vhdr, and .vmrk.
I asked BrainProduct and they said there is no difference between .eeg and .dat.
Is it possible to add .dat as an acceptable extension for BrainVision format? 
@choldgraf @dorahermes @ftadel

### PR DESCRIPTION
**Pull request by JegouA**
As discussed in the issue #172, I suggest changing the name BrainVision data format in BrainVision Core file data format in iEEG and EEG specifications.

closes #172

### CODE REVIEW

**Comment by sappelhoff** [comment]
also, I believe that according to our new rules as listed in [DECISION-MAKING](https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md), you have to add yourself to this file: https://github.com/bids-standard/bids-specification/blob/master/CODEOWNERS

Finally, after this PR is merged, you can make a new PR adding yourself as a contributor to BIDS in this file: https://github.com/bids-standard/bids-specification/blob/master/src/99-appendices/01-contributors.md

Sorry if that seems like a lot of overhead for the tiny changes. But perhaps you can see it as an opportunity and entry point for future contributions instead :-)

**Comment by chrisfilo** [comment]
My reading of the rules was that adding yourself to 01-contributors.md and CODEOWNERS is optional, but if one chooses to they need to add their name to both (not just 01-contributors.md). Maybe we need to clarify this language.

**Comment by JegouA** [comment]
Ok, I'm lost :). Do I have to add myself as a contributor to have my request accepted? Or could it be accepted anyway?

**Comment by sappelhoff** [comment]
I just re-read the `DECISION-MAKING.md` file and yes - adding oneself as a contributor is optional ... but if one decides to do that, one also needs to add oneself as a CODEOWNER.

for the present issue: @JegouA it would be completely fine for you to incorporate my feedback and then we can merge this PR in a few days (if at least one more BIDS person agrees that this is helpful)

File: src/04-modality-specific-files/03-electroencephalography.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ stored in one of the following formats:
 
 -   [European data format](https://www.edfplus.info/) (`.edf`)
 
--   [BrainVision data format](https://www.brainproducts.com/productdetails.php?id=21&tab=5)
+-   [BrainVision Core file data format](https://www.brainproducts.com/productdetails.php?id=21&tab=5)
```
Comment (sappelhoff) [inline src/04-modality-specific-files/03-electroencephalography.md]:
```suggestion
-   [BrainVision Data Exchange Core Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5)
```

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ stored in one of the following formats:
 
 -   [European Data Format](https://www.edfplus.info/) (`.edf`)
 
--   [BrainVision data format](https://www.brainproducts.com/productdetails.php?id=21&tab=5)
+-   [BrainVision Core file data format](https://www.brainproducts.com/productdetails.php?id=21&tab=5)
```
Comment (sappelhoff) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md]:
```suggestion
-   [BrainVision Data Exchange Core Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5)
```

**Comment by sappelhoff** [comment]
@JegouA do you see the two suggestions I made to your code? If you accept them, we are ready to merge this.

I suggest to use `BrainVision Core Data Format`, because it corresponds with Brain Products website: https://www.brainproducts.com/productdetails.php?id=21&tab=5

PS: They have updated their examples, and there is no mention of `.dat` anymore :-)

**Comment by sappelhoff** [comment]
Thanks!

Please feel free to make a new Pull Request to add yourself as a contributor (if you want)!

That is, you can add your name here: [01-contributors.md](https://github.com/bids-standard/bids-specification/blob/master/src/99-appendices/01-contributors.md)

**Comment by JegouA** [comment]
@sappelhoff sorry, I didn't know that I have to accept the changes

**Pull request merged**